urbs.py

```diff
@@ -964,7 +973,17 @@ def res_process_capacity_rule(m, sit, pro):
     return (m.process.loc[sit, pro]['cap-lo'],
             m.cap_pro[sit, pro],
             m.process.loc[sit, pro]['cap-up'])
-
+            
+# used roof area <= roof area
+def res_process_roof_area_rule(m, sit, pro):
+    for p in m.pro_tuples:
+        try:
```

Comment:
The `try`-`except` block should be removed once the constraint is only declared for the relevant processes...